Pyjion/absint.cpp

```diff
@@ -2312,7 +2312,49 @@ JittedCode* AbstractInterpreter::compile_worker() {
                     auto two = stackInfo[stackInfo.size() - 2];
 
                     // Currently we only optimize floating point numbers..
-                    if (one.Value->kind() == AVK_Float && two.Value->kind() == AVK_Float) {
+                    if (one.Value->kind() == AVK_Integer && two.Value->kind() == AVK_Float) {
+                        _ASSERTE(m_stack[m_stack.size() - 1] == STACK_KIND_OBJECT);
+                        _ASSERTE(m_stack[m_stack.size() - 2] == STACK_KIND_VALUE);
+
+                        if (byte == BINARY_AND || byte == INPLACE_AND || byte == INPLACE_OR || byte == BINARY_OR ||
```

Comment:
If the following `if` statement is just the rest of the possible operations it would be better to make this an `else` branch on the other `if` statement.